drivers/storage/portworx/portworx.go

```diff
@@ -69,6 +71,12 @@ func (p *portworx) Init(k8sClient client.Client, recorder record.EventRecorder)
 	return nil
 }
 
+func (p *portworx) UpdateDriver(info *storage.UpdateDriverInfo) error {
+	p.zoneToInstancesMap = info.ZoneToInstancesMap
+	p.cloudProvider = info.CloudProvider
```

Comment:
We could convert this into Cloudops cloud provider type for future use. Can do it later when you start using it.